### PR TITLE
General styling for website

### PR DESCRIPTION
**Pull request by LindseyWeiskopf**
In this branch, I made several styling changes to this branch including:
-Change of font across body text and headers
-Change of background color for body and content
-Added borders for images in the main page and in the gallery page
-Added padding to body text to make it more readable
Hopefully, these changes will make the website more aesthetically pleasing. Thanks!

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,39 +4,41 @@
...
     </div>
-
-    <div id="links">
+    <p></p>
+    <div id="content">
       <p>Find me on <a href="https://github.com/LindseyWeiskopf">GitHub</a> and <a href="https://www.linkedin.com/in/lindsey-weiskopf">LinkedIn</a>.</p>
     </div>
-    
-    <div id="random-message">
+    <p></p>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:34]: Are you inserting paragraph tags to create spacing? You can group all of this into a single paragraph tag that will likely have the same effect. And if that's not enough space, try adding a class and increasing the line space via css.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -4,39 +4,41 @@
...
     <div id="introduction">
       <h1>Hey, I'm Lindsey.</h1>
       <h2>It's nice to meet you.</h2>
-      <p>This is me.</p>
+      <h2>This is me.</h2>
       <img src="images/profilepic.JPG"/>
     </div>
-
-    <div id="projects">
+    <p></p>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:18]: Similar comment as below. I'd suggest create the desired spacing via css.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -4,39 +4,41 @@
...
-    <div id="random-message">
+    <p></p>
+    <div id="content">
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
-   
-    <div>
+   <p></p>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:40]: Same comment as above

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -4,39 +4,41 @@
...
       <h1>Hey, I'm Lindsey.</h1>
       <h2>It's nice to meet you.</h2>
-      <p>This is me.</p>
+      <h2>This is me.</h2>
       <img src="images/profilepic.JPG"/>
     </div>
-
-    <div id="projects">
+    <p></p>
+    <div id="content">
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:19]: Hmm. I noticed you use the same id name for multiple chunks of code. Ids should be unique to single code blocks. For what you're doing, classes are better. see: https://css-tricks.com/the-difference-between-id-and-class/

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (LindseyWeiskopf) [reply portfolio/src/main/webapp/index.html:34]: I fixed this by adding a top and bottom margin for all of the div tags  and got rid of the empty paragraph tags.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (LindseyWeiskopf) [reply portfolio/src/main/webapp/index.html:19]: I changed all of the non-unique ids to classes and reflected that in style.css!

**Comment by LindseyWeiskopf** [comment]
I believe that I resolved all of the issues above and pushed the new version!

**Pull request merged**